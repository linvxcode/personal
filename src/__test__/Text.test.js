import React from "react";
import { render, screen } from "@testing-library/react";
import Text from "../Layout/Text";

// test('Render Corectly', () => {
//     render(<Text />);
//     const element = screen.getByText('Welcome To My Portofolio');
//     expect(element).toBeInTheDocument();
// })

describe('Text', () => {
    it('Render Text', () => {
        const {Texttes} = render(<Text />);
        expect(Texttes).toMatchSnapshot();
        const element = screen.getByText('Welcome To My Portofolio')
        expect(element).toBeInTheDocument();
    })
}) 