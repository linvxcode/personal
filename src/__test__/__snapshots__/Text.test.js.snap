// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Text Render Text 1`] = `undefined`;
